website/api/controllers/view-endpoint-ops.js

```diff
@@ -1,25 +1,31 @@
 module.exports = {
 
 
-  friendlyName: 'View endpoint ops',
+  friendlyName: 'View compliance',
```

Comment:
the three official landing pages are endpoint ops, vulnerability managment, and device management